test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -1539,16 +1542,16 @@ public void testGetDocContentSCVerifyGroupPush() throws Exception {
     RecordingDocIdPusher pusher = new RecordingDocIdPusher();
     adaptor.init(new MockAdaptorContext(config, pusher));
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Keep an eye out for when you actually use this ByteArrayOutputStream. RecordingResponse has one by default. When we want to make assertions about it, we tend to pass one in rather than cast getOutputStream to a ByteArrayOutputStream, but if we don't need reference it at all, you can just use the built-in one.

Reply:
ok.